services/src/main/java/org/keycloak/protocol/oidc/mappers/AcrLoaProtocolMapper.java

```diff
@@ -42,17 +42,16 @@
 /**
  * @author <a href="mailto:mposolda@redhat.com">Marek Posolda</a>
  */
-public class AcrProtocolMapper extends AbstractOIDCProtocolMapper implements OIDCAccessTokenMapper, OIDCIDTokenMapper, TokenIntrospectionTokenMapper, EnvironmentDependentProviderFactory {
-
-    private static final Logger logger = Logger.getLogger(AcrProtocolMapper.class);
+public class AcrLoaProtocolMapper extends AbstractOIDCProtocolMapper implements OIDCAccessTokenMapper, OIDCIDTokenMapper, TokenIntrospectionTokenMapper, EnvironmentDependentProviderFactory {
+    private static final Logger logger = Logger.getLogger(AcrLoaProtocolMapper.class);
 
     private static final List<ProviderConfigProperty> configProperties = new ArrayList<>();
 
     static {
-        OIDCAttributeMapperHelper.addIncludeInTokensConfig(configProperties, AcrProtocolMapper.class);
+        OIDCAttributeMapperHelper.addIncludeInTokensConfig(configProperties, AcrLoaProtocolMapper.class);
     }
 
-    public static final String PROVIDER_ID = "oidc-acr-mapper";
+    public static final String PROVIDER_ID = "oidc-acr-loa-mapper";
```

Comment:
I don't think we can change this as AFAIK this will break backwards compatibility for the people migrating from the previous version as they already have `oidc-acr-mapper` in their DB.